commands/version/__tests__/version-command.test.js

```diff
@@ -232,6 +232,20 @@ describe("VersionCommand", () => {
     });
   });
 
+  describe("--scope", () => {
+    // TODO: This test makes no sense because of collect-updates mock
+    it("should change versiosn only for scoped packages", async () => {
```

Comment:
Just for some inline context, this mock for `collect-updates` does not currently allow to write any meaningful tests:
https://github.com/lerna/lerna/blob/master/commands/__mocks__/@lerna/collect-updates.js#L9
![image](https://user-images.githubusercontent.com/1346296/94282255-4353ed00-ff58-11ea-84df-2b56ff9b2f1a.png)